project/plugins.sbt

```diff
@@ -3,7 +3,7 @@ resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 resolvers += Resolver.jcenterRepo
 
-addSbtPlugin("uk.gov.hmrc"        %  "sbt-auto-build"         % "3.22.0")
+addSbtPlugin("uk.gov.hmrc"        %  "sbt-auto-build"         % "3.24.0")
 addSbtPlugin("uk.gov.hmrc"        %  "sbt-distributables"     % "2.5.0")
 addSbtPlugin("org.playframework"  %  "sbt-plugin"             % "3.0.5")
```

Comment:
change org.playframework.sbt-plugin to 3.0.6
also update others as per platops-pr-bot suggestion